CHECKOUT_code

```diff
@@ -44,5 +44,33 @@ git clone -b ${fms_release}   https://github.com/NOAA-GFDL/FMS
 git clone -b ${fms_release}   https://github.com/NOAA-GFDL/FMSCoupler
 git clone -b ${drivers_release}   https://github.com/NOAA-GFDL/atmos_drivers
 
+
```

Comment:
I think rather than commenting this out, maybe we can surround it by an if statement?  Just brainstorming: we could have an optional argument to this script (call the argument mom?).  If you simply execute with no argument it checks out the normal code above in lines 34-45.  If there is a "mom" argument given execute code in lines 48-71? 

@bensonr do you have an opinion on this?

Reply:
should we do something similar to the compile option and checkout files depending on the user's needs?
checkout, checkout shieldfull, checkout shiemom?

Reply:
code renovated